config/locales/en.yml

```diff
@@ -23,6 +23,7 @@ en:
       link_name: "Ruby on Rails Tutorial"
       link_name_help: "Help page"
       link_to_book1: "Link to Book"
+  Users: "Users"
```

Comment:
để là users: "Users" nhé, key i18n viết toàn bộ bằng chữ thường, chỉ có value được viết hoa thôi